.github/config/android-arm.json

```diff
@@ -0,0 +1,14 @@
+{
+  "docker_service": "build-android",
+  "platforms": [
```

Comment:
I think there might be a better name for this than "platform", it seems like just a name, and the platform is under the "target_platform" key right now (which I think should be renamed "platform")

Reply:
Chatted with Andrew offline about 'target_platform' parameter, and looks like we don't want to rename that parameter as it's being used to define an argument for gn (same name).  

I open for suggestions on how or if we want to rename 'platform' parameter.  I see three options here:

1. Keep it as-is.
2. Rename it to 'variant'.
3. Rename it to 'flavor'.

WDYT?